meAdota/settings.py

```diff
@@ -75,6 +77,13 @@
 ROOT_URLCONF = 'meAdota.urls'
 
 SOCIALACCOUNT_PROVIDERS = {
+    'github': {
+        'SCOPE': [
+            'user',
+            'repo',
+            'read:org',
+        ],
+    }
```

Comment:
These lines are not necessary code. The reason is: the user won't be able to login using his Github account.

You must remove it.